src/python_tools/tftp_transfer/client.py

```diff
@@ -49,50 +29,85 @@ def main():
         "--file",
         required=True,
         type=pathlib.Path,
-        help="path to the file to be transferred",
+        help="path to the file to be transferred. This should be an absolute path",
```

Comment:
I'm confused. Is this supposed to run on the robot or on our computers? I think this should be documented  somewhere